src/overlays/actors/ovl_Obj_Bean/z_obj_bean.c

```diff
@@ -494,7 +494,7 @@ void ObjBean_Init(Actor* thisx, PlayState* play) {
             ObjBean_Move(this);
             ObjBean_SetupWaitForPlayer(this);
 
-            ObjBean_InitDynaPoly(this, play, &gMagicBeanPlatformCol, DPM_UNK3);
+            ObjBean_InitDynaPoly(this, play, &gMagicBeanPlatformCol, DYNA_TRANSFORM_POS | DYNA_TRANSFORM_ROT_Y);
```

Comment:
Functions like `ObjBean_InitDynaPoly` should probably be updated to use `transformFlags` for their argument (instead of `moveFlag` or w/e else they use)

Reply:
Yeah I planned to do this and general cleanup of the InitDynaPoly helpers in another PR, they need it anyway

Note since then there's also https://github.com/zeldaret/oot/pull/1379 in the playground which does that clean-up